initramfs/init.sh

```diff
@@ -34,7 +34,7 @@ restart()
 	sleep 5s
 	reboot
 	modprobe sunxi_wdt || true
-	watchdog -T 1 -t 60 -F /dev/watchdog || true
+	watchdog -T 1 -t 60 -F "/dev/watchdog" || true
 	echo "Failed to reboot, shutting down instead."
```

Comment:
why did you not change my 'echo mess' that shell check was complaining about?

Mind you, I was not aware of shell check back then!

The echo -en "\n" uglyness I added to ensure we start with an empty line.
In hindsight, we should just remove the -en and \n altogether and replace it with:
```sh
echo ""
echo "======"
...
```
Reason i put it in a single command is because dmesg would print through our robot. The rootdelay parameter partially fixed this, as even the single line echo can still get interrupted.

Reply:
Not sure what you want here :P